apps/src/p5lab/GameLabVisualizationColumn.jsx

```diff
@@ -73,6 +74,11 @@ class GameLabVisualizationColumn extends React.Component {
 
   pickerPointerUp = e => {
     if (this.props.pickingLocation) {
+      if (isMobileDevice()) {
```

Comment:
Can you add a comment since this is a workaround for ios?

Reply:
Also, I wanted to record another offline conversation (with me, Suresh, Anjali):
Starting with ios 13, ipads will say they are desktop rather than mobile devices. We may end up needing to revisit this when that change happens.